manifests/charts/ai-gateway-helm/values.yaml

```diff
@@ -38,6 +38,18 @@ controller:
   podAnnotations: {}
   podSecurityContext: {}
   securityContext: {}
+  # Example of a podEnv
+  #  - key: AI_GATEWAY_STS_PROXY_URL
```

Comment:
I think this needs comment on exactly what AI_GATEWAY_STS_PROXY_URL does like when/how it's used